examples/js/exporters/GLTFExporter.js

```diff
@@ -1754,7 +1754,7 @@ THREE.GLTFExporter.prototype = {
 
 			} else if ( object.isLight ) {
 
-				console.warn( 'THREE.GLTFExporter: Only directional, point, and spot lights are supported.' );
+				console.warn( `THREE.GLTFExporter: light ${object.name} of type ${object.constructor.name}: Only directional, point, and spot lights are supported.` );
```

Comment:
We usually do this instead:
```js
console.warn( 'THREE.GLTFExporter: Only directional, point, and spot lights are supported.', object );
```

Reply:
Would that show the type though? That's what a user would need to search for.

Reply:
The warning looks in Chrome like so:

![image](https://user-images.githubusercontent.com/12612165/54526261-7df9c400-4976-11e9-89f9-153892569816.png)

I think it's okay like that. Users can still debug the object for more details.